src/show.jl

```diff
@@ -63,6 +63,16 @@ function Base.show(io::IO, attr::Attributes)
     print(io, "Attributes of ", attr.parent)
 end
 
+Base.show(io::IO, attrdict::AttributeDict) = summary(io, attrdict)
```

Comment:
How does Julia end up printing the rest of the attributes ? Is that in some sort of fallback AbstractDict printing?

Reply:
Yes, the REPL printing is this one:
https://github.com/JuliaLang/julia/blob/803f90db9195c5c72df90d8a424c7066f1a8f2ee/base/show.jl#L133

Reply:
Ah it's the 3 argument show method. Thanks, I was wondering why `@which` was sending me to a seemingly unrelated call.